nemoguardrails/embeddings/basic.py

```diff
@@ -199,6 +205,10 @@ async def _run_batch(self):
         """Runs the current batch of embeddings."""
 
         # Wait up to `max_batch_hold` time or until `max_batch_size` is reached.
+        if not self._current_batch_full_event:
+            raise Exception("self._current_batch_full_event not initialized")
+
+        assert self._current_batch_full_event is not None
```

Comment:
code suggestion from previous comment
```suggestion
        if (
            self._current_batch_full_event is None
            or self._current_batch_finished_event is None
        ):
            raise RuntimeError(
                "Batch events not initialized. This should not happen."
            )

```

Reply:
Addressed in #1383